web-admin/mode_check

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+#
+# This CGI script is called at the end of manufacture to verify that
+# devices are no longer in manufacturing mode.
+#
+MFG_PROVISIONED=0
+if [ -f /proc/environment/mfg_provisioned ] ; then
+    MFG_PROVISIONED=`cat  /proc/environment/mfg_provisioned`
+fi
```

Comment:
Can you make the MFG_PROVISIONED check look like this:
https://github.com/MinimSecure/minim-openwrt-feed/blob/master/web-landing/31_update_www_provisioned#L6

On MinimOS devices we don't have /proc/environment so we need to use fw_getenv

Reply:
sure.